docs/developer/cherry_pick.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Cherry Pick 
+sidebar_position: 9 
+---
+
+This page explains how to use shell script `hack/cherry_pick_pull.sh` in kubeedge/kubeedge repo to cherry pick the patch to release branch.
+
+
+# Cherry Pick Settings
```

Comment:
A little suggestion: `## Cherry Pick Settings` is better than `#` . This way, we can display the catogary on the right and generate links for each part like: 

![image](https://github.com/kubeedge/website/assets/48788150/66962015-fc8f-4a01-bfab-d6674c9e892b)

Reply:
Thanks for reminding, I have changed.